simulations/vip-converter/vip-converter-testnet/simulations.ts

```diff
@@ -0,0 +1,363 @@
+      expect(await RiskFund.riskFundConverter()).to.equal(RISK_FUND_CONVERTER);
+    });
+
+    it("ProtocolShareReserve Proxy should have new implementation", async () => {
+      expect(await ProxyAdmin.getProxyImplementation(PROTOCOL_SHARE_RESERVE_PROXY)).to.equal(
+        PROTOCOL_SHARE_RESERVE_NEW_IMPLEMENTATION,
+      );
+    });
+
+    it("PSR should have correct storage", async () => {
+      expect(await ProtocolShareReserve.poolRegistry()).to.equal(psrPoolRegistry);
+      expect(await ProtocolShareReserve.maxLoopsLimit()).to.equal(psrMaxLoopsLimit);
+      expect(await ProtocolShareReserve.totalAssetReserve(XVS)).to.equal(psrTotalAssetReserve);
+      expect(await ProtocolShareReserve.accessControlManager()).to.equal(ACM);
+      expect(await ProtocolShareReserve.owner()).to.equal(NORMAL_TIMELOCK);
+    });
+
+    it("PSR should have correct distribution configs", async () => {
+      const percentageDistributionConverters = [4000, 420, 192, 177, 211, 1000];
+      expect(await ProtocolShareReserve.totalDistributions()).to.equal(10);
+      for (let i = 0; i < 6; i++) {
+        expect(await ProtocolShareReserve.getPercentageDistribution(converters[i], 0)).to.equal(
+          percentageDistributionConverters[i],
+        );
+      }
+
+      expect(await ProtocolShareReserve.getPercentageDistribution(VTREASURY, 0)).to.equal(4000);
+      expect(await ProtocolShareReserve.getPercentageDistribution(VTREASURY, 1)).to.equal(4000);
+      expect(await ProtocolShareReserve.getPercentageDistribution(RISK_FUND_CONVERTER, 1)).to.equal(5000);
+      expect(await ProtocolShareReserve.getPercentageDistribution(XVS_VAULT_CONVERTER, 1)).to.equal(1000);
+    });
+
+    it("Timelock should be the owner of all converters", async () => {
+      for (let i = 0; i < 6; i++) {
+        const Converter = new ethers.Contract(converters[i], SINGLE_TOKEN_CONVERTER_ABI, provider);
+        expect(await Converter.owner()).to.equal(NORMAL_TIMELOCK);
+      }
+    });
+
+    it("Timelock should be the owner of ConverterNetwork", async () => {
+      expect(await ConverterNetwork.owner()).to.equal(NORMAL_TIMELOCK);
+    });
+
+    it("Timelock should be the owner of XVSVaultTreasury", async () => {
+      expect(await XVSVaultTreasury.owner()).to.equal(NORMAL_TIMELOCK);
+    });
+
+    it("XVSVaultTreasury should have correct state variables", async () => {
+      expect(await XVSVaultTreasury.XVS_ADDRESS()).to.equal(XVS);
+      expect(await XVSVaultTreasury.xvsVault()).to.equal(XVS_VAULT);
+    });
+
+    it("ConverterNetwork should contain all converters", async () => {
+      expect(await ConverterNetwork.getAllConverters()).to.deep.equal(converters);
+    });
+  });
+});
+
+forking(35922050, () => {
+  const provider = ethers.provider;
+  let RiskFundConverter: Contract;
+  let USDCPrimeConverter: Contract;
+  let usdt: Contract;
+  let usdc: Contract;
+  let user1: Signer;
+  let user1Address: string;
+  const amount = parseUnits("10", 18);
```

Comment:
I would be more generous to avoid issues with the minimum amount to privately convert:

```suggestion
  const amount = parseUnits("1000", 18);
```

Reply:
Done https://github.com/VenusProtocol/vips/pull/49/commits/9eb73c07ce3c22d0e06dac9f5ba757469d673f1c